cardano-node/cardano-node.cabal

```diff
@@ -22,29 +22,29 @@ library
 
   hs-source-dirs:      src
 
-  exposed-modules:     Cardano.Chairman
+  exposed-modules:     Cardano.Benchmarking.Tx.Generation
+                       Cardano.Benchmarking.Tx.NodeToNode
+                       Cardano.Benchmarking.Tx.TxSubmission
```

Comment:
Why are benchmarking modules part of the library itself?

Reply:
@erikd should I create a benchmark stanza in the cabal file?

Reply:
Yeah, I think that would make sense.

Reply:
Because these modules are used in `cardano-cli` as pointed out by @CodiePP I think it would make sense to move these to the `cardano-cli` executable and just move all remaining related commands under the benchmarking command group in the `cardano-cli`